### PR TITLE
#3 Add "order button" component

### PR DESCRIPTION
**Pull request by Alastor-git-user**
PR2 is completed, though I can't use console.log to print the changes in the select button

### CODE REVIEW

**Comment by Alastor-git-user** [comment]
Improving

File: src/app/components/orderButton/orderButton.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <select className={styles.orderButton}>
+            <option selected value="None" disabled>Order By</option>
+            <option value="Date">Date</option>
+            <option value="Title">Title</option>
+            <option value="Random">Random</option>
+        </select>
+        <input
+        className={styles.orderButton}
+        type="button"
+        onChange={(event) => {
```
Comment (well1791) [inline src/app/components/orderButton/orderButton.tsx:24]: @Alastor-git-user I don't think an `input` make sense here, please use `select` instead

File: src/app/components/orderButton/orderButton.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          setInputValue(event.target.value);
+        }}
+      />
+      </div>
+    </div> 
+  );
+}
+//event.target.value
```
Comment (well1791) [inline src/app/components/orderButton/orderButton.tsx:32]: @Alastor-git-user  no need for this comment

File: src/app/components/orderButton/orderButton.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+export default function OrderButton() {
+  const [orderButtonValue, setInputValue] = useState("");
+
+  console.log( orderButtonValue );
+  
+  return (
+    <div className={styles.orderButtonRectangle}>
+      <div className={styles.orderButtonContainer}>
+        <select className={styles.orderButton}>
```
Comment (well1791) [inline src/app/components/orderButton/orderButton.tsx:15]: @Alastor-git-user here, the solution should more like this (I think)

```tsx
<select
	className={styles.orderButton}
  onChange={(event) => {
    setInputValue(event.target.value);
  }}
>
```

**Review by well1791 — CHANGES_REQUESTED** [review]
Please, don't forget to remove the unused styles from `src/app/page.module.css`

File: src/app/components/orderButton/orderButton.tsx (hunk 1 above)
Comment (well1791) [reply src/app/components/orderButton/orderButton.tsx:24]: @Alastor-git-user don't forget to remove this input

**Review by well1791 — APPROVED** [review]
great job!! LGTM! (Looks good to me)

**Comment by Alastor-git-user** [comment]
Everything is just fine.

**Pull request merged**